lib/PuppeteerSharp.Tests/PageTests/BrowserContextOverridePermissionsTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace PuppeteerSharp.Tests.PageTests
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class BrowserContextOverridePermissionsTests : PuppeteerPageBaseTest
+    {
+        public BrowserContextOverridePermissionsTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private Task<string> GetPermissionAsync(Page page, string name)
+            => page.EvaluateFunctionAsync<string>(
+                "name => navigator.permissions.query({ name}).then(result => result.state)",
```

Comment:
format 😉 
should be `name => navigator.permissions.query({ name }).then(result => result.state)`